tuya.cpp

```diff
@@ -304,6 +379,21 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
                     
                 }
                 break;
+                case 0x0101: // off / running for Moe
+                {
+                    QString mode;
+                    if (data == 0) { mode = "off"; }
+                    if (data == 1) { mode = "heat"; }
```

Comment:
Doesn't handle the case when data > 1 and mode is empty.
Also always use `QLatin1String` for static strings.

```
  if      (data == 0) { mode = QLatin1String("off"); }
  else if (data == 1) { mode = QLatin1String("heat"); }
  else
  {
    DBG_Assert(data <= 0); // unsupported
    return;
  }
```

Reply:
Done